backends/hip-ref/ceed-hip-ref.h

```diff
@@ -116,19 +116,18 @@ CEED_INTERN int CeedVectorCreate_Hip(CeedSize n, CeedVector vec);
 
 CEED_INTERN int CeedElemRestrictionCreate_Hip(CeedMemType mtype, CeedCopyMode cmode, const CeedInt *indices, CeedElemRestriction r);
 
-CEED_INTERN int CeedElemRestrictionCreateBlocked_Hip(const CeedMemType mtype, const CeedCopyMode cmode, const CeedInt *indices,
-                                                     const CeedElemRestriction res);
+CEED_INTERN int CeedElemRestrictionCreateBlocked_Hip(CeedMemType mtype, CeedCopyMode cmode, const CeedInt *indices, CeedElemRestriction res);
```

Comment:
Might as well change `mtype`->`mem_type` and `cmode` -> `copy_mode` for consistencies sake?

Reply:
updated